src/main/resources/schemas/nhinc/common/configAdmin.xsd

```diff
@@ -8,6 +8,7 @@
             <xsd:element name="userInfo" type="nccommon:UserType" />
             <xsd:element name="configInstance" type="xsd:string" />
             <xsd:element name="authMethod" type="xsd:string" />
+			<xsd:element name="hashToken" type="xsd:string" />
```

Comment:
Optional: try to format so that the element is aligning with other.  Always use space instead of tabs